vignettes/a07_updating_rsyncrosim_scripts_to_v2.Rmd

```diff
@@ -37,78 +37,107 @@ Note that when you remove a package from your library any associated datasheets
+# results for the section above
+library(rsyncrosim)
+mySession <- session()
+myLibrary <- ssimLibrary("Changes to rsyncrosim v2.ssim", packages=c("stsim", "helloworldSpatial"))
+scenario1 <- scenario(myLibrary, scenario = "Scenario 1")
+scenario2 <- scenario(myLibrary, scenario = "Scenario 2")
+scenario3 <- scenario(myLibrary, scenario = "Scenario 3")
+myScenario <- scenario(myLibrary, "my Scenario")
+dependency(myScenario) <- "Scenario 1"
 ```
 
-The `addon` and `template` arguments have also been removed.
+`info(ssimLibrary)` no longer shows the *Package name*, *Current package version* and *Minimum package version*, and the Input and Output folder paths have been combined into the *Data folder*. See the <a href="https://syncrosim.github.io/rsyncrosim/articles/a01_rsyncrosim_vignette_basic.html#access-model-metadata" taret="_blank">Introduction to rsyncrosim</a> for details.
+
+The `template` and `addon` arguments have also been removed. You can download template libraries from <a href="https://cloud.syncrosim.com/" target="_blank">SyncroSim Cloud</a>.
 
 ## Dependencies
 
 To view the existing dependencies for a scenario, use:
 
-```{r view dependencies, eval=FALSE}
+```{r view dependencies}
 dependency(myScenario)
 ```
 
-To set dependencies for a scenario, use:
+To set dependencies for a scenario:
 
 ```{r set dependencies, eval=FALSE}
-dependency(myScenario) <- c("Scenario 1", "Scenario 2")
-```
+# Change from:
+dependency(myScenario, dependency = c("Scenario 2", "Scenario 3"))
 
-Note that this will remove any previously set dependencies.
-Also, the order in which the dependencies are listed can be important. In this example, `Scenario 1` takes precedence over `Scenario 2`.
+```
+```{r dependencies, warning=FALSE}
+# To:
+dependency(myScenario) <- c("Scenario 2", "Scenario 3")
 
+dependency(myScenario)
 
-## Multiprocessing
+```
 
-The `run()` function no longer has the `jobs` argument. Use the `core_Multiprocessing` datasheet to set the number of cores to use. Note that this will affect every scenario run.
+Note that this will remove any previously set dependencies unless the existing dependencies are also included in the vector.
+Also, the order in which the dependencies are listed can be important. In this example, `Scenario 2` takes precedence over `Scenario 3`.
 
 
-## Variable naming
+## Multiprocessing
 
-The **ID** in variable names have been changed to **Id**. It is generally safe to substitute `ID` for `Id` throughout your script, unless that affects any variables not related to SyncroSim.
+The `run()` function no longer has the `jobs` argument. Use the `core_Multiprocessing` library datasheet to set the number of cores to use. Note that this will affect every scenario run.
 
-## Core datasheets
+```{r multiprocessing, eval=FALSE}
+multiprocessing <-
+    data.frame(EnableMultiprocessing = TRUE,
+               MaximumJobs = 6,
+               EnableMultiScenario = TRUE) # optional, to run multiple scenarios in parallel
```

Comment:
I would remove this argument - users couldn't set this from the `run()` function before anyway